debian/changelog

```diff
@@ -1,6 +1,56 @@
+opx-nas-interface (5.20.3) unstable; urgency=medium
+
+  * Feature: Add interface name and ifindex to the output list when querying for stats
+  * Feature: Adding QSA support for 1G SFP media
+  * Feature: VxLAN Support
+  * Bugfix: Altered tap fd lock access to be called only after event add/del.
+  * Bugfix: Used the right mgmt VRF name macro to skip the VRF object creation in the SAI
+            for mgmt VRF.
+  * Bugfix: Avoid sending multiple set requests for speed to PAS
+  * Bugfix: Sending sequential hwport offset value from port group object to PAS
+            for setting media transciever
+  * Bugfix: Handle the mode change for lag blocked port in netlink handler
+  * Bugfix: Get the port mode before adding port to LAG in kernel
+  * Bugfix: Remove the publish of oper state just after connecting interface to NPU port
+  * Bugfix: Using steady clock for providing independent timestamps
+  * Bugfix: Unable to delete/clear L2MC on removal of port channel as VLAN member
+  * Bugfix: MAC learn disable issue in vlan
+  * Bugfix: The ability to set mac_lear_mode from CPS set for VXLAN
+  * Update: Exclude vlan dir from compilation
+  * Update: Enabled default libevent lock mechanism for multi thread access of event base.
+  * Update: Setting transceiver enable/disable in PAS based on Admin State and not Oper State
+  * Update: Adding port-group breakout mode for ethernet qsfp28 ports
+  * Update: Allow ports to be added/removed from VLAN 1 in scaled vlan mode
+  * Update: Re-apply admin state after membership changes
+  * Update: Consolidate remote endpoint event into a single event with tunnnel ID
+  * Update: Allow ports to be added/removed from VLAN 1 in scaled vlan mode
+  * Update: Use separate service for mac address allocation
+  * Update: Avoid sending multiple set requests for speed to PAS
+  * Update: Set only ingress split horizon for pv subport
+  * Update: Split horizon changes
+  * Update: MAC learn mode for physical port, LAG
+  * Update: set_speed method changed to support QSA adaptor
+  * Update: Check profile type as DDQSFP28 for lane admin up
+  * Update: Publish vlan sub interface event for regular vlan interfaces as well
+  * Update: Support to do get on VXLAN type interface
+  * Update: Send vlan sub interface transaction directly to base
+  * Update: Correctly handle bridge name when deleting vlan
+  * Update: Enable/disable interface packet drop status based on its tagged/untagged membership
+  * Update: Send sequential hwport offset value from port group object to PAS for setting media transciever
+  * Update: Handle the mode change for lag blocked port in netlink handler
+  * Update: Get the port mode before adding port to lag in kernel
```

Comment:
LAG should be capitalized throughout to be consistent with the rest of the CHANGELOG message.

Reply:
Same goes for VLAN actually